### PR TITLE
Feature for social login

### PR DESCRIPTION
**Pull request by footios**
Write details... like things that we want other people to review!

### CODE REVIEW

File: file.txt [hunk 1]
```diff
@@ -1 +1 @@
-world
+hello world
```
Comment (footios) [inline file.txt]: This is how it was supposed to be at the first place. But it would be better if you capitalize 'H'.

**Review by footios — COMMENTED** [review]
Please make the below mentioned changes and do another push.

File: file.txt [hunk 2]
```diff
@@ -1,4 +1 @@
-
-hello
-
-world
+Hello world!
```
Comment (fotistsakiris) [inline file.txt]: I would have made a comment about capitalizing 'h'. But I see you've already made the change. Congrats ;)

Comment (fotistsakiris) [reply file.txt]: Now that I think about it... why don't you capitalize all letters... please :)

**Review by fotistsakiris — CHANGES_REQUESTED** [review]
Please make the commented changes.

**Review by fotistsakiris — APPROVED** [review]
Everything looks perfect. Let's go ahead with merging.

**Pull request merged**